rules-topics/01_introduction.md

```diff
@@ -62,6 +62,56 @@ Welcome to Chronicles. Whether you've been LARPing for years, or have never seen
 tabard in your life, Chronicles welcomes you. For the newer players, here are a few tips
 to get you started
 
+### Creating a Character
+
+If you are a new player you will start with 100 Skill Points 10 Body and 5 Armor Training.  Skill Points are used to purchase skills like the ability to swing a sword or cast a spell.  You can use the Character Calculator to design your character.  
+* All characters earn 3 Skill Points per day of an event.
+* Characters Under 200 Build gain Skill Points at 2x the rate.
+* All Characters under 200 build can be recreated or rewritten at any time.
```

Comment:
Lowercase "Characters" here?  I would also suggest avoiding `at any time` and instead say, `without requiring a Spirit Forge.  If done during an event, a member of staff or plot must sign the new or modified character sheet.`  Would like to avoid instances where people rewrite their characters mid-combat or something else silly.

Reply:
Updated